### PR TITLE
Fix issue 55. Take updateAction parameters from array

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan 
@iariasleon 

@dmoranj

### CODE REVIEW

File: lib/models/visualRules.js [hunk 1]
```diff
@@ -162,8 +162,8 @@ function vr2rule(cardRule) {
                 case 'updateAttribute':
                     action.type = 'update';
                     action.parameters = {
-                        name: card.actionData.userParams.name,
```
Comment (fgalan) [inline lib/models/visualRules.js]: Entry in Changed next release file missing, viting #55

Comment (crbrox) [inline lib/models/visualRules.js]: Fixed in ee3e66a

**Comment by fgalan** [comment]
Lgtm (passing the ball to @dmoranj for final lgtm + merge)

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**